src/messages/SharedMessageBuilder.cpp

```diff
@@ -104,20 +105,9 @@ void SharedMessageBuilder::parse()
 
 bool SharedMessageBuilder::isIgnored() const
 {
-    // TODO(pajlada): Do we need to check if the phrase is valid first?
-    auto phrases = getCSettings().ignoredMessages.readOnly();
-    for (const auto &phrase : *phrases)
-    {
-        if (phrase.isBlock() && phrase.isMatch(this->originalMessage_))
-        {
-            qCDebug(chatterinoMessage)
-                << "Blocking message because it contains ignored phrase"
-                << phrase.getPattern();
-            return true;
-        }
-    }
-
-    return false;
+    return isIgnoredMessage({
+        /*.message = */ this->originalMessage_,
```

Comment:
This comment was a part of the old solution, right? Can those be removed now?

Also, I see that here we only pass the message, while in the other calls of `isIgnoredMessage` we pass all the things. I believe it's ok and we check for whether more members of `IgnoredMessageParameters` are present, right? Just willing to understand this logic better.

Reply:
Comments should not be removed, no

We only pass as many parameters as we need, with the exception of forcing us to include message as an empty string in other places because c++20 struct initialization isn't possible yet. With c++20 struct initialization we can omit .message in the other places too.